spec/controllers/graphql_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 # Useful resource: http://graphql.org/learn/serving-over-http/
 
 def parser_error_raised?(response)
-  data_is_empty = response["data"].nil?
+  data_is_empty = response.parsed_body["data"].nil?
```

Comment:
Shall we add the same change to the `hidden_field?` method in the `GraphQLAPI` module?